ext/ffi/ffi.stub.php

```diff
@@ -68,8 +71,18 @@ public static function isNull(FFI\CData $ptr): bool {}
 
 namespace FFI {
 
+final class CData {
```

Comment:
Should I backport these missing classes for PHP 8.0?

Reply:
Yeah, probably makes sense.

Reply:
alright, I'll try to collect the missing classes for another PR